Net12/WebMaze/Models/ImageViewModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebMaze.Models
+{
+    public class ImageViewModel
+    {
+        public string UserName { get; set; }
```

Comment:
use public virtual User Creater { get; set; } instead of it

Reply:
I use it in new push "Done"

Reply:
it"s named as author